Plots/Masking/NCL_mask_5.py

```diff
@@ -0,0 +1,154 @@
+    projection = ccrs.PlateCarree()
+    ax = plt.subplot(row, col, pos, projection=projection)
+    ax.coastlines(linewidths=0.5)
+    ax.add_feature(cfeature.LAND, color="green")
+    ax.add_feature(cfeature.LAKES, color="plum")
+    ax.add_feature(cfeature.OCEAN, color="blue")
+   
+    '''
+    Cartopy does not currently have a feature that separates island land from 
+    main land. There is also no feature to add 'ice shelf' data to a projection.
+    This addition would require another subset of data to specify area encompassed
+    by an ice shelf in a region. 
+    '''
+    # Create label names and define colors for the legend
+    land = mpatches.Rectangle((0, 0), 1, 1, facecolor="green")
+    lakes = mpatches.Rectangle((0, 0), 1, 1, facecolor="plum")
+    ocean = mpatches.Rectangle((0, 0), 1, 1, facecolor="blue")
+    
+    labels = ['Land ', 'Lakes', 'Ocean']
+    
+    plt.legend([land, lakes, ocean], labels,
+                loc='lower center', fontsize=14, bbox_to_anchor=(0.5, -0.20), ncol=3)
+    ''' 
+    Python allows for a better representation of labeling in legends. The NCL version
+    of this projection has the labels set in a caption on the graph and creates a colorbar
+    with each color representing a different region. This isn't necessary in Python and as
+    such was altered in this version.
+    '''
+    # Use geocat.viz.util convenience function to set titles and labels without calling several matplotlib functions
+    gvutil.set_titles_and_labels(
+        ax,
+        maintitle=title,
+        maintitlefontsize=16)
+
+# Plot first color map
+Plot(20, 1, (1,7), "land sea mask using 'atmos.nc'")
+
+# Plot second subplot
+
+def Plot2(row, col, pos, title):
+
+    """
+    Helper function to create two similar plots where subplot position
+    and title can all be customized on the same style
+    map projection.
+    
+    Args:
+        
+        row (:class: 'int'): 
+            number of rows necessary for subplotting of visualizations
+        col (:class: 'int'): 
+            number of columns necessary for subplotting 
+        pos (:class: 'int'): 
+            position of visualization in m x n subplot
+        title (:class: 'str'): 
+            title of graph in format "Title"
+      """
+   
+    ax1 = plt.subplot(row, col, pos, projection=ccrs.PlateCarree())
+    ax1.coastlines(linewidths=0.5)
+
+    # Import an NCL colormap
+    newcmp = 'magma' 
+    
+    # Contourf-plot data
+    contour = wrap_t.plot.contourf(ax=ax1, transform=ccrs.PlateCarree(),
+                    vmin = 240, vmax = 315, levels = 18, cmap = newcmp, add_colorbar=False)
+                 
+    #Create a colorbar for projection 
+    cbar = plt.colorbar(contour, ax=ax1, orientation='horizontal', shrink=0.70,
+                    pad=0.11, extendrect=True, extendfrac='auto', 
+                    ticks = [240,245,250,255,260,265,270,275,280,285,290,295,300,305,310])
```

Comment:
Can you use `range` `np.arange` or `np.linspace` here instead of typing all of the ticks out?